services/movie_session.py

```diff
@@ -0,0 +1,44 @@
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: int,
+        movie_id: id,
+        cinema_hall_id: int
+) -> None:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: None = None) -> None:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: None) -> None:

```

Comment:
The parameter `movie_session_id` should have a type hint of `int` instead of `None`. The return type should be `MovieSession` instead of `None` since the function returns a movie session instance.